prisma/schema.prisma

```diff
@@ -0,0 +1,40 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+
+model User {
+  id            Int       @default(autoincrement()) @id
+  username      String?
+  email         String?   @unique
+  password      String?
+  eventTypes    EventTypes[]
+  createdAt     DateTime  @default(now()) @map(name: "created_at")
+  updatedAt     DateTime  @updatedAt @map(name: "updated_at")
+}
+
+model EventTypes {
+  id            Int       @default(autoincrement()) @id
```

Comment:
the schema.prisma file looks like it's not formatted correctly. can you add a prettier plugin for it?